.github/workflows/build-caching-maven-samples.yml

```diff
@@ -0,0 +1,29 @@
+name: Build caching Maven sample validation
+
+on: [push]
+
+jobs:
+
+    verify-build-caching-maven-samples:
+        runs-on: ubuntu-latest
+        env:
+            LOG_MSG_EXTENSION_LOADED: "Build operation 'Gradle Enterprise mojo execution' completed"
+            LOG_MSG_EXECUTION_FAILED: "Error executing a GradleEnterpriseListener callback"
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Set up JDK 11
+              uses: actions/setup-java@v2
+              with:
+                  java-version: '11'
+                  distribution: 'adopt'
+            - name: Setup Nodejs and npm
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "14"
+            - name: Setup yarn
+              run: npm install -g yarn
+            - name: Run Maven verify
+              working-directory: ./build-caching-maven-samples
+              run: ./mvnw clean verify -Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug -Dgradle.scan.disabled=true
```

Comment:
How does this verify that build caching is correctly configured?

Reply:
it does not, build cache is disabled if no GE instance is connected

Reply:
can we remove the `-Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug` line?